metabase/metabase
# Clean up EditBar component

An internal clean up of the `EditBar` component (TypeScript, styled-components, etc.). The navigation project somehow broke the admin edit bar display and just refactoring the component fixed the problem

### To Verify

1. Open a dashboard and click the pencil icon to go into editing mode. Ensure the edit bar looks as it normally does
2. Go to `/admin/permissions/collections/root`, change a permission. Ensure the edit bar shows up and looks as it normally does

### Demo

<img width="1314" alt="CleanShot 2022-03-31 at 20 30 49@2x" src="https://user-images.githubusercontent.com/17258145/161134532-1f44ddcb-a4c7-4854-b6b4-ca152b075c1f.png">
